common/thorhelper/csvsplitter.cpp

```diff
@@ -73,7 +73,8 @@ void CSVSplitter::addTerminator(const char * text)
 
 void CSVSplitter::addEscape(const char * text)
 {
-    matcher.addEntry(text, ESCAPE);
+    if (text && *text)
```

Comment:
Is this the right fix? Seems like the user is specifically trying to set there to be "no escape character" - will this do that?

Reply:
If there was not defined "escape" at Spray or when CSV file created , then "No escape" (NULL) is the default. 
He received 'System error: -1: Duplicate entry "" added to string matcher' because he tried to add "no escape" (NULL) to the StringMatcher and it is throws an exception.  

The aim of my fix is to prevent this situation.

Reply:
Sounds like the correct fix is to ignore duplicates ...